mergify_engine/actions/merge/action.py

```diff
@@ -343,19 +344,36 @@ def _handle_merge_error(self, e, ctxt):
             return self._sync_with_base_branch(ctxt)
 
         elif e.status_code == 405:
-            ctxt.log.info(
-                "Waiting for the Branch Protection to be validated",
-                status=e.status_code,
-                error_message=e.message,
-            )
-            return (
-                None,
-                "Waiting for the Branch Protection to be validated",
-                "Branch Protection is enabled and is preventing Mergify "
-                "to merge the pull request. Mergify will merge when "
-                "branch protection settings validate the pull request. "
-                f"(detail: {e.message})",
-            )
+            check_name = self._extract_required_check(e.message)
+            if check_name:
```

Comment:
This variable is actually unused. You might want to at least log it to be sure it's correct?

Reply:
We already log the e.message and put it in the summary. We don't need this variable.